src/Illuminate/Support/helpers.php

```diff
@@ -553,6 +553,8 @@ function data_set(&$target, $key, $value, $overwrite = true)
      */
     function dd(...$args)
     {
+        http_response_code(500);
```

Comment:
It feels wrong to have it here, because you may very well use `dd` in non-HTTP context too. Think about background jobs, Console apps, etc.

However, it may be very well this does not negatively affect anything in non-HTTP environment. It certainly doesn't throw a warning, even when in CLI mode (I just tested), so I'm `~0` on this change.

Reply:
It should be no issue for a non-HTTP context. Please refer to the documentation: http://php.net/manual/en/function.http-response-code.php

```
FALSE will be returned if response_code is not provided and it is not invoked in a web server environment (such as from a CLI application). 
TRUE will be returned if response_code is provided and it is not invoked in a web server environment (but only when no previous response status has been set).
```

I assume the call will be ignored for non-HTTP context.

Reply:
http://php.net/manual/en/function.http-response-code.php

I assume the call is just being ignored for non-HTTP contexts.